common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -579,4 +582,36 @@ public void setCoinBaseUserAccountId(String accountId) {
     public String getCoinbaseUserAccountId(){
         return prefs.getString(PREFS_KEY_COINBASE_USER_ACCOUNT_ID, null);
     }
+
+    public void setCoinbaseUserWithdrawalLimitAmount(String amount){
+        prefs.edit().putString(PREFS_KEY_COINBASE_USER_WITHDRAWAL_LIMIT, amount).apply();
+    }
+
+    public String getCoinbaseUserWithdrawalLimitAmount() {
+        return prefs.getString(PREFS_KEY_COINBASE_USER_WITHDRAWAL_LIMIT, null);
+    }
+
+    public void setCoinbaseSendLimitCurrency(String currency){
+        prefs.edit().putString(PREFS_KEY_COINBASE_SEND_LIMIT_CURRENCY, currency).apply();
+    }
+
+    public String getCoinbaseSendLimitCurrency(){
+        return prefs.getString(PREFS_KEY_COINBASE_SEND_LIMIT_CURRENCY, GenericUtils.getLocaleCurrencyCode());
+    }
+
+    public void setCoinbaseUserInputAmount(String amount){
+        prefs.edit().putString(PREFS_KEY_COINBASE_USER_INPUT_AMOUNT, amount).apply();
+    }
+
+    public String getCoinbaseUserInputAmount() {
+        return prefs.getString(PREFS_KEY_COINBASE_USER_INPUT_AMOUNT, null);
+    }
+
+    public void setCoinbaseUserWithdrawalRemaining(String amount){
+        prefs.edit().putString(PREFS_KEY_COINBASE_USER_WITHDRAWAL_REMAINING, amount).apply();
+    }
+
+    public String getCoinbaseUserWithdrawalRemaining() {
+        return prefs.getString(PREFS_KEY_COINBASE_USER_WITHDRAWAL_REMAINING, getCoinbaseUserWithdrawalLimitAmount());
+    }
```

Comment:
We keep adding things to this Configuration class which will balloon it sooner or later beyond manageable. I'm thinking we should split it into module-specific configuration classes + a global one in the future.

Reply:
Yes, I was wondering the same thing.

The common module doesn't need to know about settings/configuration that are specific to other modules.  When we do make a change for module-specific vs global, we must take care not to break or lose any settings.